package/gluon-config-mode-autoupdater/i18n/de.po

```diff
@@ -13,5 +13,13 @@ msgstr ""
 msgid ""
 "This node will automatically update its firmware when a new version is "
 "available."
-msgstr "Dieser Knoten aktualisiert seine Firmware automatisch, sobald "
+msgstr ""
+"Dieser Knoten aktualisiert seine Firmware automatisch, sobald "
 "eine neue Version vorliegt."
+
+msgid ""
+"Automatic updates are disabled. They can be enabled in <em>Advanced "
+"Settings</em>"
+msgstr ""
+"Automatische Updates sind deaktiviert. Unter <em>Erweiterte "
+"Einstellungen</em> kannst du diese aktivieren."
```

Comment:
Klingt seltsam. Besser: "Sie können in den *Erweiterten Einstellungen* aktiviert werden."